packages/react-magma-dom/src/components/CharacterCounter/CharacterCounter.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import styled from '../../theme/styled';
+import { FormFieldContainerProps } from '../FormFieldContainer';
+import { I18nContext } from '../../i18n';
+import { InputMessage } from '../Input/InputMessage';
+import { magma } from '../../theme/magma';
+
+export interface CharacterCounterProps
+  extends Omit<FormFieldContainerProps, 'fieldId'>,
```

Comment:
Do we need `Omit<FormFieldContainerProps, 'fieldId'>,`?

Reply:
It's a required prop that Character Counter doesn't need correct?

Reply:
What this is saying is CharacterCounterProps will be `CharacterCounterProps + FormFieldContainerProps - fieldId`. But CharacterCounterProps doesn't need to extend FormFieldContainerProps because it doesn't need any of the FormFieldContainerProps like actionable, containerStyle, isLabelVisuallyHidden, etc.